### PR TITLE
✅ Tests/increase test coverage

### PR DESCRIPTION
**Pull request by Thomas-Smets**

### CODE REVIEW

**Review by j-vp — COMMENTED** [review]
intermediate comments, more review to follow

File: src/test/12_UniswapV2PricingModule.t.sol [hunk 1]
```diff
@@ -235,6 +248,11 @@ contract AssetManagement is UniswapV2PricingModuleTest {
 
         //Then: Asset is added to the Pricing Module
         assertTrue(uniswapV2PricingModule.inPricingModule(address(pairSnxEth)));
+        assertEq(uniswapV2PricingModule.assetsInPricingModule(0), address(pairSnxEth));
```
[the remark this answers is not in the document]
Comment (Thomas-Smets) [reply src/test/12_UniswapV2PricingModule.t.sol:251]: setRiskVariablesForAsset() and if the riskVars() are correctly stored is already tested in AbstractPricingModuleTest.
Don't think we have to test that as well for each of the Pricing Modules that inherit from the abstract contract.

**Review by j-vp — APPROVED** [review]
gtg

**Comment by Thomas-Smets** [comment]
Will wait until PR in Arcadia Lending for test coverage (https://github.com/arcadia-finance/arcadia-lending/pull/27) is also approved, to immediately update the libs.

**Pull request merged**